pkg/errors/errors.go

```diff
@@ -0,0 +1,8 @@
+package comboerrors
```

Comment:
Non blocking nit: This should just be `errors`. One can derrive that it contains a list of errors related to the combo project from the fact that the package exists in the combo repository.